frontend/src/components/landing/Plans.module.scss

```diff
@@ -26,13 +26,14 @@
   @include text-body-sm;
   color: $color-blue-50;
   font-weight: 700;
-  padding: $spacing-xs 0;
+  padding: $spacing-md 0;
 }
 
 .plan {
+  justify-content: space-between;
```

Comment:
This actually helps to spread out the content in the pricing plan module, which is especially needed for the case of the waitlist CTA updated plan UI where the description would otherwise leave a large gap.
https://user-images.githubusercontent.com/13066134/175574754-caeb124a-f5e2-49d6-abe4-62b4d54d3541.mov

Reply:
Huh... I'm not sure what I was thinking with my original comment, since the element certainly _is_ `display: flex` - it's set two lines below this. Sorry for the false alarm then!